Source/Managers/LuaMan.cpp

```diff
@@ -885,17 +883,17 @@ LuaMan::~LuaMan() {
 }
 
 const std::vector<std::string>* LuaMan::DirectoryList(const std::string& path) {
-	std::string fullPath = System::GetWorkingDirectory() + g_PresetMan.GetFullModulePath(path);
+	std::string fullPath = System::GetWorkingDirectory() + path;
 	auto* directoryPaths = new std::vector<std::string>();
 
-	if (IsValidModulePath(fullPath)) {
+	if (fullPath.find("..") == std::string::npos) {
```

Comment:
Doesn't this still allow absolute directory access? I.e C:/system32

Reply:
Nah, cause `fullPath` is `std::string fullPath = System::GetWorkingDirectory() + path;`, so it is forced to start with the game's directory.